datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaBasedConnectorTask.java

```diff
@@ -438,11 +438,15 @@ public void run() {
         }
       }
       postShutdownHook();
-      _stoppedLatch.countDown();
+      countDownStoppedLatch();
       _logger.info("{} stopped", _taskName);
     }
   }
 
+  protected void countDownStoppedLatch() {
```

Comment:
This is a weird api to expose to subclasses. How would an implr know when it is safe to invoke this?